### ISSUE TITLE
Bug: Complain about non empty rules block 

### ISSUE THREAD

**Issue by pke** [shinnn/vscode-stylelint#171]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

A bug where it claims an empty rule set, when it is not empty but contains dynamic prop replacement.

![image](https://user-images.githubusercontent.com/164625/38709759-034670f6-3ebd-11e8-95b7-987491c10c37.png)

Stylelint does not complain on the command line about this.

> Which rule, if any, is this issue related to?

not sure, but something like `no-emtpy-ruleset` or "no-empty-blocks"?

> What CSS is needed to reproduce this issue?
```js
<style jsx>{`
  .hero {
    background-image: url(${image});
  }
`}</style>
```

> What stylelint configuration is needed to reproduce this issue?

I am using the default configs:

"stylelint-config-standard": "^18.2.0",
"stylelint-config-styled-components": "^0.1.1",
"stylelint-processor-styled-components": "^1.3.1",
    
> Which version of stylelint are you using?

"stylelint": "^9.2.0",

> How are you running stylelint: CLI, PostCSS plugin, Node API?

The error is shown in combination with [`style-jsx`](https://github.com/zeit/styled-jsx)

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

Yes, it's related to style-jsx"

> What did you expect to happen?

"No warnings to be flagged."

> What actually happened (e.g. what warnings or errors you are getting)?

"The following warnings were flagged:"

![image](https://user-images.githubusercontent.com/164625/38709759-034670f6-3ebd-11e8-95b7-987491c10c37.png)

**Comment by shinnn** [comment]
https://github.com/shinnn/vscode-stylelint/issues/58#comment-283300513
